samples/common/include/samples/common/ethernet.h

```diff
@@ -0,0 +1,4 @@
+#ifndef __GOLIOTH_INCLUDE_GOLIOTH_ETH_H__
+#define __GOLIOTH_INCLUDE_GOLIOTH_ETH_H__
+void eth_connect(void);
+#endif
```

Comment:
```suggestion
#define __GOLIOTH_INCLUDE_GOLIOTH_ETH_H__

void eth_connect(void);

#endif /* __GOLIOTH_INCLUDE_GOLIOTH_ETH_H__ */
```

Reply:
Removing this file per your other advice.